src/patternfly/components/ChipGroup/chip-group-action.hbs

```diff
@@ -0,0 +1,6 @@
+<button class="pf-c-chip-group__action{{#if chip-group-action--modifier}} {{chip-group-action--modifier}}{{/if}}"
```

Comment:
Could this be a container for the close all button instead, then we just use a plain button in that, so that we don't have to replicate the button component styles for this element?